app/routers/api/paths.py

```
@@ -146,49 +146,44 @@ def list_paths(skip: int = 0, limit: int = 100, db: Session = Depends(get_db)):
-            if "not_found" not in unique_volumes:
-                unique_volumes["not_found"] = []
-            unique_volumes["not_found"].append(path_str)
-        except PermissionError:
-            logger.exception(f"Permission denied accessing path {path_str}")
-            if "error" not in unique_volumes:
-                unique_volumes["error"] = []
-            unique_volumes["error"].append((path_str, "Permission denied"))
+        except (FileNotFoundError, PermissionError) as e:
+            msg = "Permission denied" if isinstance(e, PermissionError) else "Path not found"
+            if isinstance(e, PermissionError):
+                logger.exception(f"Permission denied accessing path {path_str}")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)